.github/workflows/build.yml

```diff
@@ -113,12 +115,16 @@ jobs:
           - false
         include:
           - build: nightly
+            clippy: false
             experimental: true
+          - clippy: true
     steps:
       - uses: actions/checkout@v4
       - uses: Swatinem/rust-cache@v2
         with:
           workspaces: services
+      - run: echo "NO_CLIPPY=true" >> $GITHUB_ENV
+        if: github.event_name != 'schedule' && ! matrix.clippy
```

Comment:
```suggestion
        if: github.event_name != 'schedule' && !matrix.clippy
```

Reply:
While this is probably fine, all the samples on docs.github.com show a space so I'd prefer to leave it. Troubleshooting pipelines is too time-consuming.